wiki/tests/testdata/models.py

```diff
@@ -8,3 +8,7 @@ class CustomUser(AbstractUser):
         some_field = models.IntegerField(default=0)
 except ImportError:
     pass
+
+
+class CustomGroup(models.Model):
```

Comment:
Wondering a bit here... does it make sense to have a custom group with no fields? I'm just a bit surprised that it actually works :)